### DISCUSSION TITLE
Can this model be used in C ++? Or does this model support openvino.?

### DISCUSSION THREAD

**Discussion by problemSolvingProgramming** [snakers4/silero-models#132] [Q&A] [answered]
Can this model be used in C ++? Or does this model support openvino.?

**Comment by snakers4** [comment, marked as answer]
Hi,

Which particular model?

In general PyTorch has `libtorch` and a C++ API.
Some models have `onnx` versions, it may have as well.

As for openvino - when starting the project I looked into different model hubs - torch hub, tf hub, onnx models, hugging face and openvino hub. 

In general all of them just import academic artefacts pre financed by business or academic sources and provide little to no benefit to independent authors. In case of onnx models, openvino and huggingface this was more egregious, especially when they enforce little to no references to authors in really visible places.

In case of torch hub, we got some publicity out of it, so we decided to stick to it.

**Comment by rigngsbase** [comment]
@snakers4 How can I use this model in C ++? I tried a lot but unfortunately I did not succeed.

**Reply by snakers4** [reply]
https://pytorch.org/cppdocs/
